cmd/serverless-init/cloudservice/helper/gcp.go

```diff
@@ -124,7 +124,7 @@ func getSingleMetadata(httpClient *http.Client, url string) string {
 	req.Header.Add("Metadata-Flavor", "Google")
 	res, err := httpClient.Do(req)
 	if err != nil {
-		log.Error("unable to get the requested metadata, defaulting to unknown")
+		log.Info("unable to get the requested metadata, defaulting to unknown")
```

Comment:
is it safe to expose some information about what the actual error is?

Reply:
yeah this is only if the metadata internal endpoint from google is not available, we don't want to block the exection, but we will be unable to tag telemetry properly (no projectId, no containerId, no region)